docs/tutorial.rst

```diff
@@ -204,6 +204,47 @@ here is an array using Zstandard compression, level 1::
     >>> z.compressor
     Zstd(level=1)
 
+Getting the diagnostics on the compression ratio :
+
+    >>> z.info
+    Type               : zarr.core.Array
+    Data type          : int32
+    Shape              : (10000, 10000)
+    Chunk shape        : (1000, 1000)
+    Order              : C
+    Read-only          : False
+    Compressor         : Zstd(level=1)
+    Store type         : zarr.storage.KVStore
+    No. bytes          : 400000000 (381.5M)
+    No. bytes stored   : 299348530 (285.5M)
+    Storage ratio      : 1.3
+    Chunks initialized : 100/100
```

Comment:
I think this section should be in a [code block](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code).